Gems/ROS2/Code/Source/Camera/ROS2CameraSensorEditorComponent.cpp

```diff
@@ -30,8 +30,8 @@ namespace ROS2
 
     ROS2CameraSensorEditorComponent::ROS2CameraSensorEditorComponent(
         const SensorConfiguration& sensorConfiguration, const CameraSensorConfiguration& cameraConfiguration)
-        : m_sensorConfiguration(sensorConfiguration)
-        , m_cameraSensorConfiguration(cameraConfiguration)
+        : m_sensorConfiguration(AZStd::move(sensorConfiguration))
+        , m_cameraSensorConfiguration(AZStd::move(cameraConfiguration))
```

Comment:
the `AZStd::move` is not needed since the parameter is a `const&`.
So a copy will be made regardless

Reply:
Reverted the change, thanks for noticing.